OpenRA.Game/Network/Connection.cs

```diff
@@ -133,6 +132,7 @@ public sealed class NetworkConnection : EchoConnection
 		TcpClient tcp;
 		IPEndPoint endpoint;
 		readonly List<byte[]> queuedSyncPackets = new List<byte[]>();
+		readonly ConcurrentQueue<byte[]> awaitingAckPackets = new ConcurrentQueue<byte[]>();
```

Comment:
Ok, so it turns out that these are normal packets that are waiting on an Ack before being processed, not Ack packets that are waiting on something else. We should rename this to something like `sentPackets` with a comment like `// Hold our own packets until they are acknowledged by the server`?

Reply:
Addressed in #19632 now.